conf/messages.en

```diff
@@ -1491,7 +1491,7 @@ individualBuyerNinoNumber.yes.conditional.error.invalid = Enter a National Insur
 howWasDisposed.title = How was the land or property disposed of?
 howWasDisposed.heading = How was {0} disposed of?
 howWasDisposed.option1 = Sold
-howWasDisposed.option2 = Transferred out
+howWasDisposed.option2 = Transferred out to another scheme
```

Comment:
There are some tests that are currently cross-checking the label of the radio button with some enum values.
Because of this some tests have failed.
If this change is really needed, some developers would need to change the tests too.